content/en/docs/concepts/security/service-accounts.md

```diff
@@ -0,0 +1,241 @@
+`Authorization: Bearer <token>` header to the HTTP request. The API server
+checks the validity of the bearer token as follows:
+
+1. Check the token signature.
+1. Check whether the token has expired.
+1. Check whether the token is currently valid.
+1. Check the audience claims.
+
+For tokens issued using the `TokenRequest` API, the API server also checks that
+the specific Pod using the ServiceAccount still exists, matching by the
+{{< glossary_tooltip term_id="uid" text="unique ID" >}} of that Pod. For legacy
+tokens that are mounted as Secrets in Pods, the API server checks the token
+against the Secret.
+
+For more information about the authentication process, refer to
+[Authentication](/docs/reference/access-authn-authz/authentication/#service-account-tokens).
+
+### Authenticating service account credentials in your own code {authenticating-in-code}
+
+If you have services of your own that need to validate Kubernetes service
+account credentials, you can use the following methods:
+
+* [TokenReview API](/docs/reference/kubernetes-api/authentication-resources/token-review-v1/)
+  (recommended)
+* OIDC discovery
+
+The Kubernetes project recommends that you use the TokenReview API, because this
+method invalidates tokens that are bound to API objects such as Secrets,
+ServiceAccounts, and Pods when those objects are deleted. For example, if you
+delete the Pod that contains a projected ServiceAccount token, the TokenReview
+API invalidates that token immediately. If you use OIDC validation instead, your
+clients continue to treat the token as valid until the token reaches its
+expiration timestamp.
+
+## Alternatives
+
+* Issue your own tokens using another mechanism, and then use
+  [Webhook Token Authentication](/docs/reference/access-authn-authz/authentication/#webhook-token-authentication)
+  to validate bearer tokens using your own validation service.
+* Provide your own identities to Pods.
+    * [Use the SPIFFE CSI driver plugin to provide SPIFFE SVIDs as X.509 certificate pairs to Pods](https://cert-manager.io/docs/projects/csi-driver-spiffe/).
+    * [Use a service mesh such as Istio to provide certificates to Pods](https://istio.io/latest/docs/tasks/security/cert-management/plugin-ca-cert/).
+* Authenticate from outside the cluster to the API server without using service account tokens:
+    * [Configure the API server to accept OpenID Connect (OIDC) tokens from your identity provider](/docs/reference/access-authn-authz/authentication/#openid-connect-tokens).
+    * Use service accounts or user accounts created using an external Identity
+      and Access Management (IAM) service, such as from a cloud provider, to
+      authenticate to your cluster.
+* [Configure the kubelet to retrieve credentials from an image registry](/docs/tasks/administer-cluster/kubelet-credential-provider/).
+* Use a Device Plugin to access a virtual Trusted Platform Module (TPM), which
+  then allows authentication using a private key.
```

Comment:
+ CSR API with client certs, possibly?

Reply:
OK, but IMO not having it here shouldn't block approval. The perfect is the enemy of the published.

Reply:
Added with a link to https://kubernetes.io/docs/tasks/tls/managing-tls-in-a-cluster/